SwanHeader/style/base.css

```diff
@@ -0,0 +1,86 @@
+#swan-header {
+  --swan-color-dark-1: #09263c;
+  --swan-color-dark-2: #0053a1;
+  --swan-color-dark-3: #2c7bb6;
+  --swan-color-dark-4: #00385c;
+  --swan-color-orange: #f60;
+  --swan-color-grey: #dadada;
+  --swan-border-radius: 1px;
+
+  height: 40px !important;
+  min-height: 40px;
+}
+
+.sw-header .sw-swan-logo {
+  width: 40px;
+  height: 40px;
+  background-image: url(swan_cloud.png);
```

Comment:
Wondering if we should take the logo gives by `/hub/logo` instead...

Reply:
I'm not sure how to do this, webpack tries to parse any relative URLs (/hub/logo) I add here and errors with file not found.

Reply:
Not sure either.. Leave it. But the best would be to take it from there, **if exists**, or from your local asset.